com.microsoft.mrtk.graphicstools.unity/Runtime/Experimental/Magnifier/Scripts/MagnifierManager.cs

```diff
@@ -172,7 +178,27 @@ public void ApplyMagnification()
         {
             Shader.SetGlobalFloat(MagnificationPropertyName, 1.0f - Magnification);
         }
-
+        public void GetMousePos()
+        {
+            if (inLensMode)
+            {
+                Vector3 wldpos = Camera.main.ScreenToViewportPoint(Input.mousePosition);
+                Vector4 Mousewldpos = new Vector4(wldpos.x, wldpos.y, 0, 0);
+                Shader.SetGlobalVector("_MousePos", Mousewldpos);
+            }
+            else
+            {
+                Shader.SetGlobalVector("_MousePos", new Vector4(0.5f,0.5f,0,0));
+            }   
+        }
+        private void Update()
```

Comment:
nit: Newline needed.
```suggestion

        private void Update()
```